src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryText.cs

```diff
@@ -219,7 +219,14 @@ public static FormulaValue Text(EvalVisitor runner, EvalVisitorContext context,
                     {
                         // It's a number, formatted as date/time. Let's convert it to a date/time value first
                         var newDateTime = Library.NumberToDateTime(runner, context, IRContext.NotInSource(FormulaType.DateTime), new NumberValue[] { num });
-                        resultString = ExpandDateTimeExcelFormatSpecifiers(formatString, "g", newDateTime.Value, culture, runner.CancellationToken);
+                        try
+                        {
+                            resultString = ExpandDateTimeExcelFormatSpecifiers(formatString, "g", newDateTime.Value, culture, runner.CancellationToken);
+                        }
+                        catch (FormatException)
+                        {
+                            return CommonErrors.GenericInvalidArgument(irContext, StringResources.Get(TexlStrings.ErrTextInvalidFormat, culture.Name));
```

Comment:
>return CommonErrors.GenericInvalidArgument(irContext, StringResources.Get(TexlStrings.ErrTextInvalidFormat, culture.Name)); [](http://example.com/codeflow?start=28&length=123)

This portion is not being reach by test coverage. Can we have the following test case added:
```
>> Text(30470.01953125, "yyyy-mm-dd hh:mm:ss.00000000")
Error({Kind:ErrorKind.InvalidArgument})
```

Reply:
Added